Python/pywarpx/picmi.py

```diff
@@ -1951,6 +1962,8 @@ def init(self, kw):
         self.file_prefix = kw.pop('warpx_file_prefix', None)
         self.file_min_digits = kw.pop('warpx_file_min_digits', None)
         self.dump_rz_modes = kw.pop('warpx_dump_rz_modes', None)
+        self.particle_fields_to_plot = kw.pop('warpx_particle_fields_to_plot', None)
```

Comment:
This can default to an empty dictionary, `{}`, avoiding the check of `is not None` below.

Reply:
With this version, this can be an empty list, `[]`, for the same reason.